db/db.go

```diff
@@ -1050,3 +1056,81 @@ func (db database) GetUserAssignedOrganizations(pubkey string) []OrganizationUse
 	db.db.Where("owner_pub_key = ?", pubkey).Find(&ms)
 	return ms
 }
+
+func (db database) AddBudgetHistory(budget BudgetHistory) BudgetHistory {
+	db.db.Create(&budget)
+	return budget
+}
+
+func (db database) CreateOrganizationBudget(budget BountyBudget) BountyBudget {
+	db.db.Create(&budget)
+	return budget
+}
```

Comment:
both of these functions do the exact same thing, we shouldn't have duplicate code

Reply:
No, they don't. the BudgetHistory and BountyBudget are different tables, I will rename the variables to make it clearer.

Reply:
oh my bad I must have read it wrong